app/Models/MenuLink.php

```diff
@@ -169,6 +169,11 @@ public function getRouteParams()
                 $parameters['all_members'] = 1;
             }
         }
+
+        if ($this->menu && isset($this->options['subview_filter'])) {
```

Comment:
The previous if already checks if it's going to a submenu, so you can add the second section of the if in the previous if block to make the code easier to read ;-)
```php
if (!empty($this->menu)) {
   if ($this->menu == 'all-members') {
   }
   if (isset($this->options['subview_filter'])) {
   }
}
```